test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpExporterOptionsExtensionsTests.cs

```diff
@@ -131,5 +131,70 @@ public void AppendPathIfNotPresent_TracesPath_AppendsCorrectly(string inputUri,
 
             Assert.Equal(expectedUri, resultUri.AbsoluteUri);
         }
+
+        [Fact]
+        public void AppendExportPath_EndpointNotSet_EnvironmentVariableNotDefined_NotAdjusted()
+        {
+            ClearEndpointEnvVar();
+
+            var options = new OtlpExporterOptions { Protocol = OtlpExportProtocol.HttpProtobuf };
+
+            options.AppendExportPath(options.Endpoint, "test/path");
+
+            Assert.Equal("http://localhost:4317/", options.Endpoint.AbsoluteUri);
+        }
+
+        [Fact]
+        public void AppendExportPath_EndpointNotSet_EnvironmentVariableDefined_Adjusted()
+        {
+            ClearEndpointEnvVar();
```

Comment:
```suggestion
```

The clearing of the environment is unnecessary as it is immediately set to a value

Reply:
Removed.